test/e2e/util_gatewayapi_test.go

```diff
@@ -947,3 +950,48 @@ func (m *vapManager) enable() {
 		m.t.Errorf("failed to find vap %q: %v", m.name, err)
 	}
 }
+
+func eventuallyClusterRoleContainsAggregatedPolicies(t *testing.T, destClusterRoleName, srcClusterRoleName string) error {
+	t.Helper()
+
+	return wait.PollImmediate(time.Second, timeout, func() (bool, error) {
```

Comment:
Yeesh, we have multiple tests using this `timeout` const, which is defined here: https://github.com/openshift/cluster-ingress-operator/blob/e60422dca3964c4f00b7cac64944d5ec2a874baa/test/e2e/configurable_route_test.go#L27

So we have multiple unrelated tests using the same generically named const, which is defined in an arbitrary location, with an arbitrary value, with no godoc.  We should clean that up at some point.  

I don't mean to pick on this PR though.  Evidently we've been cargo-culting this `timeout` for some time.  At some point, we should clean all these uses of `timeout` up.